packages/adblocker/src/engine/bucket/network.ts

```diff
@@ -162,4 +162,25 @@ export default class NetworkFilterBucket {
 
     return this.badFiltersIds.has(filter.getId());
   }
+
+  public getHtmlModifiers(
+    request: Request,
+    isFilterExcluded?: (filter: NetworkFilter) => boolean,
+  ): NetworkFilter[] {
+    const filters: NetworkFilter[] = [];
+
+    this.index.iterMatchingFilters(request.getTokens(), (filter: NetworkFilter) => {
```

Comment:
This looks like we iterate though all filters. But we probably also do it for the regular network filters already. It may be a significant performance problem.

Reply:
Added html index. :+1: